cdap-ui/app/cdap/components/AbstractWidget/SchemaEditor/RowButtons/FieldAttributes/RecordEnumAttributes.tsx

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import * as React from 'react';
+import WidgetWrapper from 'components/ConfigurationGroup/WidgetWrapper';
+import Button from '@material-ui/core/Button';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import { IAttributesComponentProps } from 'components/AbstractWidget/SchemaEditor/EditorTypes';
+
+const useStyles = makeStyles({
+  root: {
+    marginTop: '15px',
+    maxHeight: '250px',
+    overflowY: 'auto',
+    '& >div': {
+      margin: '10px 0',
+    },
+  },
+});
+
+function RecordEnumTypeAttributes({
+  typeProperties,
+  onChange,
+  handleClose,
+}: IAttributesComponentProps) {
+  const [doc, setDoc] = React.useState(typeProperties.doc || '');
+  const [aliases, setAlias] = React.useState(typeProperties.aliases || []);
+  const classes = useStyles();
+
+  const onChangeHandler = () => {
+    onChange('typeProperties', {
+      doc,
+      aliases,
+    });
+    handleClose();
+  };
+  return (
+    <React.Fragment>
+      <div className={classes.root}>
+        <WidgetWrapper
+          pluginProperty={{
+            name: 'doc',
+            macroSupported: false,
+            description: 'doc for the record',
```

Comment:
Nit: Should this be capitalized for consistency?

Reply:
I actually got feedback for this even before creating the PR and yet missed this one 🙄 . Thanks for catching this!.